src/index.js

```diff
@@ -304,16 +294,34 @@ export class ULabel {
         $(document).on("click", "#" + ul.config["toolbox_id"] + ' a.tbid-opt', (e) => {
             let tgt_jq = $(e.currentTarget);
             let pfx = "div#tb-id-app--" + ul.state["current_subtask"];
-            let crst = ul.state["current_subtask"];
+            let subtask_key = ul.state["current_subtask"];
+            let current_subtask = ul.subtasks[subtask_key];
```

Comment:
```suggestion
            const subtask_key = ul.state["current_subtask"];
            const current_subtask = ul.subtasks[subtask_key];
```
You're only ever reading these variables, so it's best practice to make them constants.